table/index.go

```diff
@@ -38,7 +38,7 @@ type Index interface {
 	// Meta returns IndexInfo.
 	Meta() *model.IndexInfo
 	// Create supports insert into statement.
-	Create(ctx sessionctx.Context, rm kv.RetrieverMutator, indexedValues []types.Datum, h int64, opts ...*CreateIdxOpt) (int64, error)
```

Comment:
There are some options for `Create`, it is better to make `ss kv.Transaction` as an option.

Reply:
Updated